assets/javascripts/discourse/templates/admin-wizards-transfer.hbs

```diff
@@ -0,0 +1,34 @@
+<h3>Select Wizards to export</h3>
+{{log this}}
+
+<form {{action 'export' wizards on="submit"}} class="form">
+  <ul>
+    {{#each model as |w|}}
+
+      <li style="list-style-type: none;">
+        {{input type="checkbox" id=(dasherize w.id) change=(action 'checkChanged')}}
+
+        {{#link-to "adminWizard" (dasherize w.id)}}{{w.name}}{{/link-to}}
+      </li>
+    {{/each}}
+
+  </ul>
+  {{!--  <button type='submit' class='btn btn-primary'>--}}
+  {{!--    {{ i18n 'admin.wizard.transfer.export' }}--}}
+  {{!--  </button>--}}
+  {{d-button label="admin.wizard.transfer.export" action=(action "export")}}
+
+
+</form>
+
+{{input id='file_url' type="file" change=(action "setFilePath")}}
+<br />
```

Comment:
Generally speaking we don't create space using \<br>. You can add space in using CSS in the plugin stylesheet.